# Goal
# <empty>
---
# Result
bool done
---
# Feedback
float32 xmin
float32 xmax
float32 ymin
float32 ymax
bool target_found
